azure/utils.go

```diff
@@ -62,3 +62,9 @@ func toLower(_ context.Context, d *transform.TransformData) (interface{}, error)
 	valStr := types.SafeString(d.Value)
 	return strings.ToLower(valStr), nil
 }
+
+func idToAccountName(ctx context.Context, d *transform.TransformData) (interface{}, error) {
+	id := types.SafeString(d.Value)
+	account_name := strings.Split(id, "/")[8]
```

Comment:
Don't use snake case to define a variable, use camel case instead.
```suggestion
	accountName := strings.Split(id, "/")[8]
```